.aspell.en.pws

```diff
@@ -87,3 +87,4 @@ vBucket
 vBuckets
 virtualised
 backoffs
+resynchronise
```

Comment:
This spell check flag-up is legit - we use American English in the docs.

Reply:
ah okay I'll revert that change and use the american version